packages/katello/rubygem-katello/rubygem-katello.spec

```diff
@@ -4,19 +4,26 @@
 %global foreman_min_version 1.18.0
 %global plugin_name katello
 %global gem_name katello
-# %%global prever .rc1
+%global prerelease .rc1
 %global mainver 3.8.0
-%global release 1.nightly
+%global release 1
+
+%global gem_instdir %{gem_dir}/gems/%{gem_name}-%{version}%{?prerelease}
```

Comment:
These macros are part of `rh-ruby24-rubygem-devel` and you shouldn't need to duplicate them.